playbooks/aws_role/undeploy.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
```

Comment:
Be careful with this whole undeploy block. Can you check with ansible if there is another function/action related with this role?